views/about.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- page title -->
+    <title>About</title>
+    <!-- link to ABOUT css file -->
+    <link rel="stylesheet" href="/public/styles/stylesAbout.css" />
+    <!-- Font families   -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <!-- giving an icon to the browser tab -->
+    <link rel="icon" href="/public/images/icon.png" />
+  </head>
+  <body>
+    <header>
+      <div class="navbar-container">
+        <img
+          src="/public/images/tarantino-favicon.png"
+          alt="Tarantino's face"
+          id="tarantino-favicon"
+        />
+
+        <nav class="button-navbar-container">
+          <button type="button" class="navbar-buttons">
+            <a class="navbar-a" href="/views/home.html">Home</a>
```

Comment:
On the `a` tag, you don't want and you can't send the user for your file itself. You want to redirect them to `/films` or `/photogallery` etc. Pages (views) wich you are handling on your `app.js`. 
So, it needs to be like this:
`<a class="navbar-a" href="/home.html">Home</a>`